### DISCUSSION TITLE
Drillings/Cuts at two opposite faces

### DISCUSSION THREAD

**Discussion by adrimg89** [cwapi3d/cwapi3dpython#218] [Q&A]
Hello everybody.

I am working at a CLT project and I need to know if I have some panels with cuts or drillings at two faces of the panel.
I have seen all the threads at this documentation, but I didn't find anything about this issue.

I have many doubts about how to know all the machine works that affect each panel.
When I need to know it at manual program, I usually went to: Export>Sheets>Breakdown (I'm not sure if this is the correct name, at spanish software is called "despiece")>Calculate  --> Where is this function at api doc?

and then: Export>Sheets>Breakdown>ElaborationControl

Is there any function where I can analyze that elaborations and where are they at the panel?

Thank you very much in advance.

Kind regards

**Comment by Brunner246** [comment]
Hi, 
Please upload either some screenshots or an 3d file (zipped) to visualize the issue.

**Reply by adrimg89** [reply]
Hi! 
Thank you for your reply.

I'm sorry, maybe I didn't explain it very well.

The issue is that I don't know how to analyze, using API python, all cuts or drillings of a panel, and their position respect of its faces.
Are there any API functions for calculating that machine's work?

Thank you so much.

**Comment by ancelin-charpentier** [comment]
Hey @adrimg89 ,
I might not provide the answer you need as it is not using Python. 
However it seems like it is what you are looking for. 
Have you tried a visual check while in viewer mode? It is suitable if your project is not very big.
For bigger project you should export the DPP of your CLT panels in 2d, then isolate all your dashed lines and it should make it very easy to spot the blind holes.
I hope that helps.

**Reply by adrimg89** [reply]
Hi @ancelin-charpentier, thank you very much for your reply!
That sounds like a very good idea!!
I will try it!
